packages/core/src/animation/AnimatorStateMachine.ts

```diff
@@ -15,10 +16,28 @@ export class AnimatorStateMachine {
    * @remarks When the Animator's AnimatorController changed or the Animator's onEnable be triggered.
    */
   defaultState: AnimatorState;
+  /** @internal */
+  _entryTransitions: AnimatorStateTransition[] = [];
+  /** @internal */
+  _anyStateTransitions: AnimatorStateTransition[] = [];
 
   /** @internal */
   _statesMap: AnimatorStateMap = {};
 
+  /**
+   * The list of entry transitions in the state machine.
+   */
+  get entryTransitions(): Readonly<AnimatorStateTransition[]> {
+    return this._entryTransitions;
+  }
+
+  /**
+   * The list of AnyState transitions.
+   */
+  get anyStateTransitions(): Readonly<AnimatorStateTransition[]> {
```

Comment:
```ts
 /**
   * The list of any state transitions.
   */
```

Reply:
AnyState is a special state not  any state